src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -61,14 +62,20 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
         try {
             return $this->handleRaw($request, $type);
         } catch (\Exception $e) {
-            if (false === $catch) {
-                $this->finishRequest($request, $type);
+        } catch (\Throwable $e) {
```

Comment:
should be removed: we don't want to deal with \Error in HttpKernel::handle
OR, if someone wants to discuss this statement, let's do it in a dedicated PR